### PR TITLE
fix: autoresponse triggers again

### PR DESCRIPTION
**Pull request by Chicken**
:/ forgot that triggers consisting of two words exists

### CODE REVIEW

**Review by KommentatorForAll — APPROVED** [review]
...

File: src/modules/autoresponses/textResponses.js [hunk 1]
```diff
@@ -15,10 +15,7 @@ export async function messageCreate(client, message) {
             const triggers = responseDto.trigger.some((trigger) =>
                 trigger instanceof RegExp
                     ? message.content.toLowerCase().match(trigger)
-                    : message.content
-                          .toLowerCase()
-                          .split(' ')
-                          .some((w) => w === trigger)
+                    : message.content.toLowerCase().match(new RegExp(`\\b${trigger}\\b`))
```
Comment (WilsontheWolf) [inline src/modules/autoresponses/textResponses.js:18]: Do we want to filter out any regex data in here? As these are not regex triggers its assumed we don't want to trigger regex in them. A user might put something such as a ^ and now its trying to check if its the begining.

Comment (WilsontheWolf) [reply src/modules/autoresponses/textResponses.js:18]: See: https://stackoverflow.com/questions/3446170/escape-string-for-use-in-javascript-regex

Comment (Chicken) [reply src/modules/autoresponses/textResponses.js:18]: A valid point but what are the chances if this ever happening?

**Review by WilsontheWolf — APPROVED** [review]

**Pull request merged**